bitnami/apache/values.yaml

```diff
@@ -315,6 +315,11 @@ ingress:
   ## @param ingress.apiVersion Force Ingress API version (automatically detected if not set)
   ##
   apiVersion: ""
+  ## IngressClassName is used to reference the IngressClass that should be used to implement this Ingress.
```

Comment:
Could you add @param IngressClassName? 
We have a tool to generate the README.md with these notations

Reply:
Sorry for that, just added the @param.